R/breaks.r

```diff
@@ -59,8 +62,70 @@ log_breaks <- function(n = 5, base = 10) {
     if (max == min) return(base^min)
 
     by <- floor((max - min) / n) + 1
-    base^seq(min, max, by = by)
+    breaks <- base ^ seq(min, max, by = by)
+    relevant_breaks <- base ^ rng[1] <= breaks & breaks <= base ^ rng[2]
+    if (sum(relevant_breaks) >= (n - 2)) return(breaks)
+
+    # the easy solution to get more breaks is to decrease 'by'
+    while (by > 1) {
+      by <- by - 1
+      breaks <- base ^ seq(min, max, by = by)
+      relevant_breaks <- base ^ rng[1] <= breaks & breaks <= base ^ rng[2]
+      if (sum(relevant_breaks) >= (n - 2)) return(breaks)
+    }
+    log_sub_breaks(rng, n = n, base = base)
+  }
+}
+
+#' Calculate intermediate log-scale breaks
```

Comment:
Needs `@noMd` since we're not exporting.

Reply:
The `@md` / `@noMd` tags turn Markdown parsing on / off for the given block. See https://cran.r-project.org/web/packages/roxygen2/news.html.

Is there another tag to remove the help file? Or maybe render is to a `non-exported` help file?

Reply:
Uh, typo, sorry - I meant `@noRd`

Reply:
done in e043d03